src/components/interactive-map.js

```diff
@@ -290,11 +312,14 @@ export default class InteractiveMap extends PureComponent {
         ref: this._eventCanvasLoaded,
         style: eventCanvasStyle
       },
-        createElement(StaticMap, Object.assign({}, this.props, {
-          visible: this.checkVisibilityConstraints(this.props),
-          ref: this._staticMapLoaded,
-          children: this._eventManager ? this.props.children : null
-        }))
+        createElement(StaticMap, Object.assign({}, this.props,
+          this._transitionManger && this._transitionManger.getTransionedViewport(),
```

Comment:
Is it ok if `getTransionedViewport()` returns `null` ?

Reply:
Yes, Object.assign works with null and undefined.